BackEnd/utils/administration/send_email_feedback.py

```diff
@@ -0,0 +1,42 @@
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import render_to_string
+from django.conf import settings
+
+from utils.images.send_email import set_admin_email
+
+EMAIL_CONTENT_SUBTYPE = "html"
+
+
+def send_email_feedback(user_email, message, category):
+    """Function for sending feedback letters to the administrator and the user."""
+    context = {
+        "category": category,
+        "message": message,
+        "user_email": user_email,
+    }
```

Comment:
The variable 'protocol', which is used in constructing the URL for the img tag with the CraftMerge logo in the email, is missing, which might cause it not to be displayed."

Reply:
Done! Thank you